README.md

```diff
@@ -119,42 +119,54 @@ All designs are in [Figma](https://www.figma.com/files/team/834568130405102661/H
   - ```nvm use `cat .nvmrc` ```
 
 Optional:
-- `VSCode`, with `eslint` and `prettier` plugins
+- `VS Code`, with [recommended extensions](./.vscode/extensions.json).
+  - While this is optional, it is worth noting that it is in use by the whole development team
+  - Using VS Code lets us share [default configuration settings](./.vscode/default.settings.json) and easily run scripts using [VS Code tasks](./.vscode/tasks.json).
 - `docker`
+  - Docker is necessary in order to create images for the [deploy process](https://hms-dbmi.atlassian.net/wiki/spaces/GL/pages/3009282049/Deployment)
+  - It is also used to run a local instance of the application when using the test scripts in the `./etc` directory
 
 ### Development
 
 After checking out the project, cd-ing into it, and setting up a Python 3.9 virtual environment,
-- Get `app.conf` from another developer and place it at `context/instance/app.conf`.
+- Get `app.conf` from [Confluence](https://hms-dbmi.atlassian.net/wiki/spaces/GL/pages/3045457929/app.conf) or from another developer and place it at `context/instance/app.conf`.
 - Run `etc/dev/dev-start.sh` to start the webpack dev and flask servers and then visit [localhost:5001](http://localhost:5001).
+  - If using VS Code, you can also use the `dev-start` task, which will launch these services in separate terminal windows.
 
 You will see an warning about `Cannot find source file '../src/index.ts'`, but just ignore it; [Issue filed](https://github.com/hubmapconsortium/portal-ui/issues/1489).
```

Comment:
I don't think I've seen this happen since we integrated TypeScript into the application; can we drop this line?

Reply:
@john-conroy Could you confirm whether you've seen this error in the past month or two?

Reply:
Not that I'm aware of, but I can re-run the script to double check.